nilearn/plotting/displays.py

```diff
@@ -695,6 +695,9 @@ def _map_show(self, img, type='imshow',
         to_iterate_over = zip(self.axes.values(), data_2d_list)
         for display_ax, data_2d in to_iterate_over:
             if data_2d is not None:
+                if LooseVersion(matplotlib.__version__) == LooseVersion("2.1.0"):
+                    if data_2d.min() is np.ma.masked:
+                        continue
```

Comment:
I wonder whether this is not a strategy that we could use always (ie for all matplotlib version). If the data is fully masked, there is no point trying to plot anything.

Also, could you add a comment that explains what the if does? It's not obvious to people, I guess.